very easy/24.js

```diff
@@ -1,14 +1,14 @@
 function isPlural(word) {
-	
+  return [...word][word.length - 1] == "s";

```

Comment:
I don't understand using that spread operator. Could you explain me that?

Reply:
i thought that the "length" property is only available for spread operator - changed